Fix failing build

Badges added in https://github.com/erming/shout/pull/465 emphasize that the Travis CI build is currently failing. This is a friendly reminder to fix that.

Hum, are you sure @floogulinc?

The error on https://travis-ci.org/erming/shout/builds/49741250 says:
> Error: Cannot find module '/home/travis/.shout/config'

I'm not saying your commit would not fix another issue on Travis CI, but this one doesn't seem related to your fix.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,6 @@
 language: node_js
 node_js:
   - "0.10"
+before_script:
+  - mkdir ~/.shout
+  - cp defaults/config.js ~/.shout

